wire-library/wire-runtime-swift/src/test/swift/CodableTests.swift

```diff
@@ -309,6 +309,32 @@ extension CodableTests {
 }
 
 
+// MARK: - Duration and Timestamp
+
+extension CodableTests {
+    struct DurationAndTimestamp: Equatable, Codable {
+        var duration: Wire.Duration
+        var timestamp: Wire.Timestamp
+    }
+
+    func testDurationRoundtrip() throws {
```

Comment:
Can we add a test for a duration larger than 2^53 seconds, and one for `0.1s` and make sure they round trip?

We could add similar ones for Timestamp and comment them out too, just to illustrate that that doesn't work right now (with a comment)

Reply:
https://github.com/square/wire/pull/2380/commits/e12a65fbe024131a4962b84e95575bad0130c20c